offline/ci.sh

```diff
@@ -101,16 +116,16 @@ calling_charts=(
 )
 
 # wire_version=$(helm show chart wire/wire-server | yq -r .version)
-wire_version="4.39.0"
+wire_version="4.39.159"
 
 # same as prior.. in most cases.
-wire_calling_version="4.39.0"
+wire_calling_version="4.39.159"
 
 # TODO: Awaiting some fixes in wire-server regarding tagless images
 HELM_HOME=$(mktemp -d)
 export HELM_HOME
 
-helm repo add wire https://s3-eu-west-1.amazonaws.com/public.wire.com/charts
+helm repo add wire https://s3-eu-west-1.amazonaws.com/public.wire.com/charts-develop
```

Comment:
aha. this is smart. rewrite this file a bit, so this is the way we do this now?

Reply:
The clean solution would be to extract parameters for the parts that depend on the version. And, fill with with what we get from Stefan M.'s `build.json` files: https://github.com/wireapp/wire-builds/blob/108f7eb93bc6d1227f04c23032fab0c58390e5fe/build.json

This refactoring doesn't have to be done now. However, it's good to keep in mind that this changing of versions / Helm repos buried deep inside some files is neither the goal nor the future.

Reply:
this was temporary change to test the changes needed in helm charts for k8s 1.28
Goal is to use the build.json later.